### PR TITLE
Fix death splash

### LINKED ISSUE #1025: [BUG] Death/win splash not being displayed
## 🐛 Bug Report
The match report should always be displayed, whether winning or losing.

### PR DESCRIPTION
**Pull request by BertovDev**
This PR fixes the Final death Splash.

The problem:
- When the match ends and you were in espectate mode. Final splash wat not showing correctly.

Be careful!
There is 2 death screen. One when you get kill and the final when the match ends. Make sure that the last one is working correctly. It should show you the winner player. If you win or lose.

closes #1025

### CODE REVIEW

File: client/Assets/Scripts/UI/DeathSplash/DeathSplashManager.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ void OnEnable()
         ShowMessage();
         ShowMatchInfo();
         ShowPlayerAnimation();
-        ShowEndGameScreen();
+        // ShowEndGameScreen();
```
Comment (AngieDutra) [inline client/Assets/Scripts/UI/DeathSplash/DeathSplashManager.cs:79]: If this is not being used, I would delete it

File: client/Assets/Scripts/UI/DeathSplash/DeathSplashManager.cs [hunk 2]
```diff
@@ -174,30 +174,20 @@ private void ShowPlayerAnimation()
     public void ShowEndGameScreen()
     {
         // TODO: get image from lobby
-        if (SocketConnectionManager.Instance.GameHasEnded())
+        print(SocketConnectionManager.Instance.GameHasEnded());
```
Comment (AngieDutra) [inline client/Assets/Scripts/UI/DeathSplash/DeathSplashManager.cs:177]: I believe you forgot a print here

**Review by AngieDutra — CHANGES_REQUESTED** [review]

**Review by AngieDutra — DISMISSED** [review]
Tested in mobile and working great!

File: client/Assets/Scripts/CustomLevelManager.cs [hunk 3]
```diff
@@ -57,6 +57,8 @@ public class CustomLevelManager : LevelManager
 
     private bool deathSplashIsShown = false;
 
+    public bool test = false;
```
Comment (agustinesco) [inline client/Assets/Scripts/CustomLevelManager.cs:60]: I think we should delete this

Comment (BertovDev) [reply client/Assets/Scripts/CustomLevelManager.cs]: Done

**Review by ncontinanza — APPROVED** [review]
LGTM! Tested in Unity and Android

**Review by agustinesco — APPROVED** [review]
LGTM! the death splash seems to be working correctly, tested both unity and android versions

**Pull request merged**